sklearn/multioutput.py

```diff
@@ -568,7 +569,12 @@ def predict_proba(self, X):
             else:
                 X_aug = np.hstack((X, previous_predictions))
             Y_prob_chain[:, chain_idx] = estimator.predict_proba(X_aug)[:, 1]
-            Y_pred_chain[:, chain_idx] = estimator.predict(X_aug)
+            if self.chain_method == "predict":
+                Y_pred_chain[:, chain_idx] = estimator.predict(X_aug)
+            elif self.chain_method == "predict_proba":
+                Y_pred_chain[:, chain_idx] = estimator.predict_proba(X_aug)
```

Comment:
Yes, `predict_proba` returns a 2d array, and I had mentioned this in the issue comment. This was just a mock implementation, to know whether I was going on the right path.